### PR TITLE
FEAT: Add Yi-9B

### LINKED ISSUE #1114: FEAT: Support Yi-9B
The Yi series models are the next generation of open-source large language models trained from scratch by [01.AI](https://01.ai/).
9B model is the best at coding and math in the Yi series models.

### PR DESCRIPTION
**Pull request by mujin2**
Fixes #1114 
add Yi-9B

### CODE REVIEW

File: xinference/model/llm/llm_family.json [hunk 1]
```diff
@@ -3753,50 +3764,5 @@
         "<|im_sep|>"
       ]
     }
-  },
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:3756]: Why do you remove gemma?

**Comment by ChengjieLi28** [comment]
Please rebase to the latest main code to run the CI.

**Comment by qinxuye** [comment]
The deletion of gemma should be reverted also.

File: xinference/model/llm/llm_family.json [hunk 2]
```diff
@@ -3804,4 +3815,4 @@
       ]
     }
   }
-]
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:3807]: Keep the newline.

File: xinference/model/llm/llm_family_modelscope.json [hunk 3]
```diff
@@ -2384,4 +2396,4 @@
       ]
     }
   }
-]
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:2387]: Ditto.

**Review by qinxuye — COMMENTED** [review]
Try not modify the format of file, like new line at the end etc.

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**